jinahub/encoders/text/AudioCLIPTextEncoder/executor/audioclip_text.py

```diff
@@ -33,6 +25,15 @@ def __init__(
         *args,
         **kwargs
     ):
+        """
+        :param model_path: path to the pre-trained AudioCLIP model.
+        :param default_traversal_paths: default traversal path (used if not specified in
+            request's parameters)
+        :param default_batch_size: default batch size (used if not specified in
+            request's parameters)
+        :param device: device that the model is on (should be "cpu", "cuda" or "cuda:X",
```

Comment:
Also, why don't we document *args and **kwargs? Not a big deal, just wanted to check it's intentional and agreed upon.

Reply:
we don't display them in the hub website
so for hub they are not needed but having them will not cause a problem
from the pep conventions, not having them in the docstrings is not considered a breach of pep rules and will not fail linting
Nan said that "adding duplicated codes will definitely increases our maintenance effort" so maybe that's an argument to not include them